src/app/hooks/useResponsive.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+
+export const useResponsive = () => {
+  const [isMobileView, setIsMobileView] = useState<boolean>(window.innerWidth < 950 ? true : false);
+  useEffect(() => {
+    const setResponsiveness = () => {
+      return window.innerWidth < 950 ? setIsMobileView(true) : setIsMobileView(false);
```

Comment:
what I meant was that you can replace `window.innerWidth < 950 ? true : false` with just `window.innerWidth < 950` since it will return a boolean